### ISSUE TITLE
Lot of errors

### ISSUE THREAD

**Issue by samotBest** [Android-PowerUser/Windroid_11#7]
Hello, i am installing this windows on SD card in termux. I use the original script of course. Then i started extracting files in termux. No OTG no PC, just SD card in mobile. After +- 5 hours it started writing lot of errors like this one: `ERROR: Can not open output file : Operation not permitted : /data/data/com.termux/files/home/storage/external-1/ubuntu-fs/user/share/man/man3/libwww : : lwptut.3pm.gz `
And it is not just one ERROR.

![Screenshot_20240321-193931](https://github.com/Android-PowerUser/Windroid_11/assets/159287670/3b111022-ca55-43d4-a239-cee24f76be36)

**Comment by Android-PowerUser** [comment]
That seems to be the manual. It should work that way too. Let me know what happens.

**Comment by samotBest** [comment]
Ok, so, it is safe to continue. But what does this Error mean? **Operation not permitted**

**Comment by Android-PowerUser** [comment]
With root, that would probably be off the table. But it's probably not that bad. We'll see.

**Comment by samotBest** [comment]
Fun fact is that my SD card have 128Gb and now, when procces 57%, my SD card have only 59GB.

**Comment by Android-PowerUser** [comment]
Was there anything else on it? If not, then it shouldn't be like that.

**Comment by samotBest** [comment]
I bought it, formated to the exFAT format, and put it in my Andoid 11 phone. But android copy some files automaticaly, look.
![Screenshot_20240321-203421](https://github.com/Android-PowerUser/Windroid_11/assets/159287670/8233a8a2-2d57-4c39-a1dc-d7f4e19867be)

**Comment by samotBest** [comment]
Maybe i need to do this? [https://support.google.com/android/thread/99771949?hl=en&msgid=99810175](url)

**Comment by Android-PowerUser** [comment]
I don't think that would be the solution. I think in this case it's because of the cluster size. Each file and folder uses as much as the size of a cluster, regardless of whether it's full or empty. With 475,000 files and a cluster size of 32 KB, that's 15.2 GB. You can try to create a cluster size as small as possible using a computer or root. You can try reading the Android/data/com.termux/ folder in parallel to see how many and how large files are there. Remember that Android/data/ is also created automatically and is essential here. Without this path, Windroid 11 fails.

**Comment by Android-PowerUser** [comment]
Nevertheless, the total size on the SD should only be 26 GB with the cluster size.

**Comment by samotBest** [comment]
On SD SD card Samsung\Android\data\com.termux\files :
![image](https://github.com/Android-PowerUser/Windroid_11/assets/159287670/2442fddb-fcf8-4306-bdbd-0c79d603dea7)
then in ubuntu-fs: 
![image](https://github.com/Android-PowerUser/Windroid_11/assets/159287670/399e1c33-faa4-4337-8135-3a9977df55ad)


Cluster size, is it this? 
![image](https://github.com/Android-PowerUser/Windroid_11/assets/159287670/863bc8ad-772b-42d1-a5de-f04683559def)

**Comment by Android-PowerUser** [comment]
I can't see any sizes on the map. I can't tell if it will complete successfully. Yes, those in the picture are cluster sizes.

**Comment by samotBest** [comment]
Ok, i will wait to complete. Goodbay

**Comment by samotBest** [comment]
Finished, and broken. 
![Screenshot_20240322-071404](https://github.com/Android-PowerUser/Windroid_11/assets/159287670/814d5d7a-930e-48ea-8569-9207189de470)

If i open new sesion and type ./Win11 :
![Screenshot_20240322-071443](https://github.com/Android-PowerUser/Windroid_11/assets/159287670/dad7c0fa-12e1-44f2-a857-5634a1655d30)

**Comment by Android-PowerUser** [comment]
If you have root, try `su` and then try `./Win11` again. Otherwise I don't know what to do. I will adjust the documentation a bit.

**Comment by samotBest** [comment]
Ok, i Will reinstall all, reformat the SD, And try again. Anyway, thank you

**Comment by Android-PowerUser** [comment]
I guess a reinstall is not helpful. The Android shell don't support 7z or PRoot. Root is also not helpful. I will remove the microSD install script.

**Comment by Android-PowerUser** [comment]
I have here a fork with the broken script. https://github.com/Android-PowerUser/Windroid_11/blob/Windroid-11-SD-Card-unfinished/README.md

**Comment by samotBest** [comment]
Ok, anyway this Is amazing projekt.

**Comment by Android-PowerUser** [comment]
Thanks 😃😅

**Comment by samotBest** [comment]
@Android-PowerUser 
Can you please send me the script again? I will try it again anyway. I fully reinstall termux. Maybe the problem was permissions, wrong version. Maybe SD type XC not supported. Who knows? I always try everything twice if it not work.

**Comment by samotBest** [comment]
Ok great, thank you. So, which cluster type should I choose? 16 KB?

**Comment by Android-PowerUser** [comment]
As small as possible.

**Comment by samotBest** [comment]
Hey, i just found the problem, maybe :D.  i see that termux go to SD card by this code `cd ./storage/external-1/` right?
BUT, i write some file on SD card and then type this `cd ./storage/external-1/` , then i type `ls` and there is nothing!!!
By the way, is it normal that i have external-0 and external-1 ?

**Comment by Android-PowerUser** [comment]
Anything you save to external-1 goes into `/Android/data/com.termux/files/` in the microSD. You might not see anything because you saved to the root directory of them. I've never seen anyone have external-1 and 0. As long as 1 is physically accessible there shouldn't be a problem. Otherwise you can type termux-setup-storage and you'll get the new symlinks. Files won't be deleted.

**Comment by samotBest** [comment]
Thats strange, okey.
![Screenshot_20240325-201454](https://github.com/Android-PowerUser/Windroid_11/assets/159287670/6cd20944-d65c-4cd4-87f8-af5ad68d45f8)

**Comment by Android-PowerUser** [comment]
🤔Maybe external-0 is the internal storage on the same path... I don't have media-0 and 1 at all.

**Comment by TEAMBNO** [comment]
![Screenshot_20240519_124448_com termux](https://github.com/Android-PowerUser/Windroid_11/assets/88188472/0a020917-177d-4d5d-9a6f-6a68a3ea1aeb)
Why is this happening?

**Comment by Android-PowerUser** [comment]
Win11.7z consists of 3 files! 

.001 .002 and .003 

You must have them in the same folder. Download all 3 files! They will be downloaded automatically with the code. Do not rename them.

**Comment by TEAMBNO** [comment]
They are all in the Download file

**Comment by TEAMBNO** [comment]
![IMG_20240519_130712_703](https://github.com/Android-PowerUser/Windroid_11/assets/88188472/bde58672-a05f-4824-9af9-62f6e8d829ac)

**Comment by Android-PowerUser** [comment]
Where is Win11.7z.003? Don't rename. Download all 3 files. https://github.com/Android-PowerUser/Windroid_11/releases/download/2023.07.29/Win11.7z.003 is missing.

**Comment by TEAMBNO** [comment]
![Screenshot_20240519_175843_com termux](https://github.com/Android-PowerUser/Windroid_11/assets/88188472/7fbc115e-c82b-494d-a7d8-199f7c66761a)
I downloaded the three files, but the file will not be decompressed

**Comment by TEAMBNO** [comment]
![IMG_20240519_180306_926](https://github.com/Android-PowerUser/Windroid_11/assets/88188472/6e794e92-a9e4-49c4-9a5f-3e4cbcd0688c)

**Comment by TEAMBNO** [comment]
I think file number 1 is corrupt

**Comment by Android-PowerUser** [comment]
checksum of Win11.7z.001

CRC32: 9171cd4a
MD5: 90f88287729f6c4afdd3e40676136d28
SHA1: 3bb768f2417266e870453a405aad9b2289e2bc54
SHA256: 27793a85e8e99689546ec39fd15978041cbdfc156fa39b3fbc67313146514966
SHA384: a1995d2e2eea824a4d1892584ab93516c84010c3547180e049daba62089c648e323908c6c85e2a0cb886c30fb2a49778
SHA512: 3961d94ddc4cb3928e98097e215daac757a8834fbe064bb0f75c2cd91963046feaf12b2140443892633b5ee8553b3d463a036ad9f31de8be647e5bc528b990e7


checksum of Win11.7z.002

CRC32: 8d9296e4
MD5: 56f42bf9c6643a7c496a292d3b01e922
SHA1: ecc3ff7c74078b49d2d67b0a5ba5ca7c699796bb
SHA256: 130f2b9467fae8bab0e0174588115ab3c3c664518877f9bd3f1b5fc6ed1af7c8
SHA384: 7d005506201a15402d6feed13792e6f05e60cfaa1c78311992681d33df12a0f99e97728d8b7ce7a95428337dc75e84c5
SHA512: 80cc951879f0a85137c4598d2d82789e432452ca3a878709a068938257d2c8c92716002034b9111ff1f15d770e3ab7595f7fbf0c32c9135b2f8030a4adb341a1

checksum of Win11.7z.003

CRC32: Unbekannt
MD5: 1718814d9b67e16ca0c61485cd859210
SHA1: d4616e8cb90a207c29a88b502c3b1788db0d9868
SHA256: 2c5f7aa59155ac98f94a03fc9abdb0007f84bf6e0f925c13f6b560570472ded9
SHA384: Unbekannt
SHA512: 3ab15ac83a9b1545df146007964e1a5a03f1a58c479473d84fb64d62dc67176a1c990e167f4ab3b064db6aec284ebbfbfd4835451c56c3e1e0e6011dff680861

**Comment by TEAMBNO** [comment]
How can I use this to decompress

**Comment by Android-PowerUser** [comment]
These are checksums to compare files.

**Comment by samotBest** [comment]
Hello @Android-PowerUser, do you tryed to repair the SD card installation issue? Two weeks ago i tried to install it on another mobile, with the same SD card, but the same problem. Maybe termux not support well 128GB sd cards?

**Comment by Android-PowerUser** [comment]
It's more of a permissions problem that can probably only be solved with root rights. Windroid 11 is one of many of my projects and is primarily a kind of feasibility study because when I started I didn't know if something like that would work at all except for the Renegade Project. Unfortunately, this is limited to a few models and the installation is more difficult and risky with dual boot, root and bootloader unlocking. Hence the alternative. When building a Win11 root version, however, I encountered problems with VNC and Perl. See dev-group https://t.me/Windroid_11

**Comment by TEAMBNO** [comment]
I will contact you on Telegram

**Comment by Android-PowerUser** [comment]
You can open a new issue instead or contact the Windroid 11 chatbot if you have any questions. It now runs on Claude 3 Haiku.

**Comment by TEAMBNO** [comment]
Is there a detailed explanation of the system's operation?

**Comment by Android-PowerUser** [comment]
It's in the Readme, the main page.

2024-05-19T18:20:38Z BNO Source ***@***.***>:
